ai-service/src/ai_service/embeddings/transformer.py

```
@@ -0,0 +1,56 @@
+    if torch.cuda.is_available():
+        return "cuda"
+
+    if hasattr(torch.backends, "mps") and torch.backends.mps.is_available():
+        return "mps"
+
+    return "cpu"
+
+
+@lru_cache(maxsize=1)
+def get_model(
+    trust_remote_code: bool = False,  # Might be required for some code models
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)